src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -50,7 +50,7 @@ export default function ProjectSnippet({
   };
 
   const handleRedirect = () => {
-    router.push(`https://donate.plant-for-the-planet.org/?to=${project.id}`);
+    router.push(`${process.env.NEXT_PUBLIC_DONATION_URL}/?to=${project.id}`);
```

Comment:
If this should be a real full feature donation link, you should do it like it is done in https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1363 (better wait for that PR to be merged, so you do not have to write the code twice and created merge conflicts).

Reply:
That's merged in this now